### PR TITLE
Update to FG 2.0

### PR DESCRIPTION
**Pull request by AbrarSyed**
- [x] setting up workspace
 - [x] FML version not required at dev time
 - [x] deobfuscation_data.lzma not required at dev time
 - [x] generating patches
 - [x] building universal
 - [x] building installer (jar)
 - [x] building installer (exe) ** still has some issues **
  - [x] removed the necessity of elevated permissions
 - [x] building userdev (untetsed and probably subject to change.. maybe)
 - [x] building changelog
 - [x] grabbing crowdin stuff
 - [x] deploying to maven (untested on the jenkins, works locally)

Yeah I need to squash, just thought i would open the PR first.

~~There are issues with the dev-time deobf part, unfortunately~~
dev-time deobf was removed from this PR.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -16,71 +11,168 @@ buildscript {
...
+apply plugin: "maven"
+apply plugin: "net.minecraftforge.gradle.patcher"
 
-import static net.minecraftforge.gradle.dev.ForgeDevPlugin.*
-
-apply plugin: 'maven'
-apply plugin: 'forgedev'
+minecraft {
+    version = "1.8"
+    //mappings = 'stable_16'
```
Comment (mbax) [inline build.gradle]: Intentional?

Comment (AbrarSyed) [inline build.gradle]: What intentional?
 - removal of the old 'maven' and 'forgedev' plugin definitions?  `yes`
 - setting the MC version to 1.8? `yes`
- the commented out 'stable_16' `no, but that doesnt really matter does it?`

Comment (mbax) [inline build.gradle]: Sorry, was pointing just to the selected line, the additional commented line :3

File: l4jManifest.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+  <!-- following stuff is so the EXE doesnt ask for admin access -->
+  <trustInfo xmlns="urn:schemas-microsoft-com:asm.v3">
+    <security>
+      <requestedPrivileges>
+        <requestedExecutionLevel level="asInvoker" uiAccess="false"/>
+      </requestedPrivileges>
+    </security>
+  </trustInfo>
+  <autoElevate>FALSE</autoElevate>
```
Comment (killjoy1221) [inline l4jManifest.xml]: Remove this. My bad.

**Comment by me4502** [comment]
@AbrarSyed - Any news on this getting done soon-ish?

**Comment by LexManos** [comment]
It sounds like ForgeDev is done/almost done. 
UserDev needs to be functional tho so we don't push out any bad Forge builds for modders.

**Comment by Andrey96** [comment]
What about merging forge and fml with this update?

**Comment by AbrarSyed** [comment]
apperantly this is in the Fg2 branch already.. so...

**Pull request closed without merging**